tests/integration/tui/test_updater.py

```diff
@@ -147,7 +147,16 @@ async def test_filters(one_conf, flow, scheduler, run, updater):
             'graph': {
                 'R1': 'a & b & c',
             }
-        }
+        },
+        'runtime': {
+            # TODO: remove this runtime section in
+            # https://github.com/cylc/cylc-flow/pull/5721
+            'root': {
+                'simulation': {
+                    'default run length': 'PT1M',
```

Comment:
The patch, short runtimes could cause tasks to go through running & succeeded even whilst the workflow is paused.